client/views/Home/User.js

```diff
@@ -18,10 +20,14 @@ class User extends DataComponent {
     }
     this.actions.github.getUser({ requestId: 'user', params: opt })
   }
-
   renderData({ user }) {
     return (<div className="ui card">
-      <img className="ui image" src={user.get('avatar_url')} />
+      <img
+        className="ui image user-image"
+        src={user.get('avatar_url')}
+        style={this.props.greyscale ? { WebkitFilter: 'grayscale(100%)' } : {}}
+        onClick={this.actions.user.toggleGreyscale}
```

Comment:
should be wrapped in a function, otherwise the `toggleGreyscale` action will receive the click event as the payload